src/content/docs/ai-monitoring/intro-to-ai-monitoring.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Introduction to AI monitoring'
+metaDescription: 'AI monitoring lets you observe the AI-layer of your tech stack, giving you a holistic overview of the health and performance of your AI-powered app.'
+freshnessValidatedDate: never
+---
+
+import aiTraceViewIntroPage from 'images/ai_screenshot-full_trace-view-intro-page.webp'
+
+import aiAIResponsesOverview from 'images/ai_screenshot-full_AI-responses-overview.webp'
+
+When New Relic talks about AI, we're referring to the layer of your environment that uses a large language model (LLM) to generate a response when it receives an end user prompt. Your app contains an AI assistant, but behind the AI are libraries, cost implications, vector databases, and new processes that can be monitored.   
```

Comment:
@akristen "When New Relic talks about AI..." feels a little odd to me. Especially since, until very recently, we often meant "applied intelligence". I think it's worth exploding this acronym at the start to "artificial intelligence".

What if you abstracted it out to something more like: "When people talk about artificial intelligence, they can mean different things. At New Relic, when we say AI, we mean the layer of your environment... etc etc"

Reply:
"Your app contains an AI assistant, but behind the AI are libraries, cost implications, vector databases, and new processes that can be monitored." What valuable about this (because I think probably people don't really need to be told this if they're interested in AI monitoring) is how you can use this to more effectively transition to AI monitoring on line 23.

For example:

"Your app contains an AI assistant, but behind the AI are libraries, cost implications, vector databases, and new processes that can be monitored.

With AI monitoring, you can measure the performance of the engine powering your AI assistant, so that you can ensure your users have the best possible experience. To get started, all you need is one of our APM agents. etc etc"